preview-tui splits are inverted

marioortizmanero opened the pull request:
I forgot to invert them properly. It now works like this:

![image](https://user-images.githubusercontent.com/25647296/84159030-df6eec80-aa6c-11ea-8ce3-a294b1e8c958.png)

![image](https://user-images.githubusercontent.com/25647296/84159056-e72e9100-aa6c-11ea-9993-602b5505243c.png)

Which is how it used to work, and how you'd expect it.

jarun commented:
Can you also add a dummy SPLIT= at the top so users know where to change?

marioortizmanero commented:
Shouldn't `SPLIT` be indicated as an environmental variable, like `SPLIT=v nnn` or with `export`? Otherwise I'd have to add `TERMINAL` to the top, too.

jarun commented:
Can we have the following:

    SPLIT="${SPLIT}"
    TERMINAL="{TERMINAL}"

jarun commented:
This is for users who do not want to export in the environment and use a fixed terminal with fixed size.

marioortizmanero commented:
What about

```sh
SPLIT=
TERMINAL=
```

or maybe

```sh
SPLIT="$SPLIT"  # you can set a permanent split here
TERMINAL="$TERMINAL"  # same goes for the terminal
```

jarun commented:
Second one is fine.

marioortizmanero commented:
Done. I also added a clarification regarding `SPLIT`, as it might be interpreted in two different ways (see tmux).

jarun commented:
Thank you!
